content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Monitoring Linux or macOS host metrics using a node exporter
+---
+
+# Monitoring Linux or macOS host metrics using a node exporter
+
+A Prometheus [**node exporter**](https://github.com/prometheus/node_exporter) exposes a wide variety of hardware- and OS-related metrics.
+
+In this guide, we will:
+
+* Start up a node exporter on `localhost`
+* Start up a Prometheus instance on `localhost` that's configured to scrape metrics from the running node exporter
+
+NOTE: While the Prometheus node exporter is for *nix systems, there is a [WMI exporter](https://github.com/martinlindhe/wmi_exporter) for Windows that serves an analogous purpose.
+
+## Installing and running the node exporter
+
+The Prometheus node exporter is a single static binary that you can install [via tarball](#tarball-installation) or using [`go get`](#go-installation). You can also install and run the node exporter as a [Docker image](#docker).
```

Comment:
Docker is not recommended, and should not be mentioned.

I'd just have the tarball, give the user one easy way to do things rather than potentially put them down the path of having a working Go environment.